docs/howto/krustlet-on-kubernetes-with-inlets.md

```diff
@@ -0,0 +1,273 @@
+# Running Krustlet on Kubernetes with inlets
+
+These are steps for running a krustlet node on your own computer. You can run any kind of 
+Kubernetes cluster you like, whether that's local on your computer or remote in managed 
+Kubernetes offering.
+
+These instructions are adapted from a [gist by Alex Ellis](https://gist.github.com/alexellis/d55d6d6a96ea9ae8d9d65b95297ec27e).
+
+## Prerequisites
+
+* [inlets - "The Cloud Native Tunnel"](https://docs.inlets.dev/)
+
+Run with `sudo` for an automatic installation, or run without sudo to download the binary to 
+your local directory.
+
+```shell
+curl -sLS https://get.inlets.dev | sudo sh
```

Comment:
We generally try to discourage `curl | sudo sh` in documentation, especially from a third-party domain we have no control over. Can you please recommend another installation path?

See lines 44-48 below for an example.

Reply:
I added a note about sudo and non sudo. Happy to invert that to the opposite. Would that be better?